packages/gitgraph-react/src/Tag.tsx

```diff
@@ -1,17 +1,16 @@
 import * as React from "react";
-import { Commit } from "@gitgraph/core";
+import { Tag } from "@gitgraph/core";
 
 interface Props {
-  name: string;
-  commit: Commit<React.ReactElement<SVGElement>>;
+  tag: Tag;
 }
 
 interface State {
   textWidth: number;
   textHeight: number;
 }
 
-export class Tag extends React.Component<Props, State> {
+export class TagElement extends React.Component<Props, State> {
```

Comment:
Necessary rename to avoid conflict with core `Tag` element (could have renamed the import, but I found `TagElement` to be a better option).